<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_video</class>
 <widget class="QDialog" name="edit_video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>518</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Editar</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="txt_id">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nome :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_name"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Tipo :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_tipo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Preço :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_price"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Faixa Etária : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_age"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ID Vídeo :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_id"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ID Alugando : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="video_idp"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="btn_image">
      <property name="text">
       <string>Alterar imagem</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_save">
      <property name="text">
       <string>Salvar Mudanças</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
